File: Client/app/src/main/java/com/example/myapplication/dataClasses/UpdateRangUser.kt
package com.example.myapplication.dataClasses

import kotlinx.serialization.Serializable

@Serializable
data class UpdateRangUser(
    val type : String,
    val objectUpdate : String,
    val dataUpdated : Int,
    val dialog_id: String,
    val tagUser: String
)

File: WorkDB/WSClientDB/dataClasses/UpdateRangUser.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WSClientDB.dataClasses
{
    class UpdateRangUser
    {
        public string dialog_id { get; set; }
        public string authorId { get; set; }
        public string tagUser { get; set; }
        public int dataUpdated { get; set; }
    }
}
